example/rbac/prometheus-operator/prometheus-operator-deployment.yaml

```diff
@@ -26,7 +26,7 @@ spec:
       containers:
       - args:
         - --kubelet-service=kube-system/kubelet
-        - --prometheus-config-reloader=quay.io/prometheus-operator/prometheus-config-reloader:v0.76.0
+        - --prometheus-config-reloader=quay.io/prometheus-operator/prometheus-config-reloader:v0.76.1
```

Comment:
Interesting!
Should it be 0.76.2?

Reply:
It should be fixed now that conflicts are resolved.